### ISSUE TITLE
Navigation entre les exercices

### ISSUE THREAD

**Issue by jacquev6** [jacquev6/Gabby#9]
Léa :
> Rajouter les boutons « Précédent » et « Suivant » pour les exos en cours de modification (pour ne pas perdre l’utilisateur, il vaut mieux que l’interface reste similaire, cela évite que l’utilisateur se trompe de boutons)

@LeaPacini Il faudra qu'on réfléchisse vers quels exercices ces boutons doivent pointer : pendant la création d'une suite d'exercices, ils fonctionnent comme un historique, donc si tu crées 3, 1, 5 puis clique sur "précédent" il te ramène au 1, et si tu cliques encore, il te ramène au 3. Si depuis la list des exercices d'une page tu cliques sur "Editer" un exercice, ce contexte historique n'existe pas. Je peux faire pointer "précédent" vers l'exercice le plus proche dans l'ordre des numéros, mais il n'existe pas toujours. Est-ce que je dois étendre cette recherche aux pages précédentes ? Bref, il y a un peu de comportement à définir.

**Comment by LeaPacini** [comment]
Effectivement, nous allons y réfléchir. Je reviens vers toi un peu plus tard à ce sujet pour être sûre de ne pas te dire de bêtises.
